spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/notify/OpsGenieNotifier.java

```diff
@@ -0,0 +1,207 @@
+package de.codecentric.boot.admin.notify;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.expression.Expression;
+import org.springframework.expression.ParserContext;
+import org.springframework.expression.spel.standard.SpelExpressionParser;
+import org.springframework.web.client.RestTemplate;
+
+import de.codecentric.boot.admin.event.ClientApplicationEvent;
+import de.codecentric.boot.admin.event.ClientApplicationStatusChangedEvent;
+
+/**
+ * Created by fernando on 2/8/17.
+ */
+public class OpsGenieNotifier extends AbstractStatusChangeNotifier {
+    public static final URI DEFAULT_URI = URI.create("https://api.opsgenie.com/v1/json/alert");
+    private static final String DEFAULT_MESSAGE = "#{application.name}/#{application.id} is #{to.status}";
+
+    private final SpelExpressionParser parser = new SpelExpressionParser();
+    private RestTemplate restTemplate = new RestTemplate();
+
+    /**
+     * BASE URL for OpsGenie API
+     */
+    private URI url = DEFAULT_URI;
+
+    /**
+     * Integration ApiKey
+     */
+    private String apiKey;
+
+    /**
+     *  Comma separated list of users, groups, schedules or escalation names
+     *  to calculate which users will receive the notifications of the alert.
+     */
+    private String recipients;
+
+    /**
+     * Comma separated list of actions that can be executed.
+     */
+    private String actions;
+
+    /**
+     * Field to specify source of alert. By default, it will be assigned to IP address of incoming request
+     */
+    private String source;
+
+    /**
+     * Comma separated list of labels attached to the alert
+     */
+    private String tags;
+
+    /**
+     * The entity the alert is related to.
+     */
+    private String entity;
+
+    /**
+     * Default owner of the execution. If user is not specified, the system becomes owner of the execution.
+     */
+    private String user;
+
+
+    /**
+     * Trigger description. SpEL template using event as root;
+     */
+    private Expression description;
+
+    public OpsGenieNotifier() {
+        this.description = parser.parseExpression(DEFAULT_MESSAGE, ParserContext.TEMPLATE_EXPRESSION);
+    }
+
+    @Override
+    protected void doNotify(ClientApplicationEvent event) throws Exception {
+        restTemplate.postForEntity(buildUrl(event), createEvent(event), Void.class);
```

Comment:
Please add a correct content-type to the request. Depending on the order of the MessageConverters it happens that the requests end up being in xml. If you need a hint doing that please have a look at the LetsChatNotifier.